shared/src/interfaces/user.interface.ts

```diff
@@ -0,0 +1,24 @@
+import { UserSex } from '../common/enums/user/usersex.enum';
+import { UserType } from '../common/enums/user/usertype.enum';
```

Comment:
```suggestion
import { UserSex } from '~/common/enums';
import { UserType } from '~/common/enums';
```
`~` - is the alias for the root shared folder. It would be better to use it

Let's do the `proxy imports` (you need to import all enums to the `common/enums/index.ts` and then use the `import` from there)